src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -4861,166 +4865,121 @@ namespace HVACUnitarySystem {
 			TotalFloorAreaOnAirLoop = 0.0;
 			AirLoopNumber = 0;
 
```

Comment:
This next section was completely revised. Needed a new way to determine if a UnitarySystem was zone, air loop, or OA Sys equipment. No need to review since you can't really see what was changed here. Basically, a section each for air loop, then OA sys, then zone equipment, tests to see if a UnitarySystem is present. Flags are set accordingly.